config/infection.json.dist

```diff
@@ -0,0 +1,30 @@
+{
+    "source": {
+        "directories": [
+            "src/Psl/"
+        ],
+        "excludes": [
+            "Async",
+            "RandomSequence",
+            "Internal",
+            "{.*/Internal/.*}"
+        ]
+    },
+    "timeout": 10,
+    "logs": {
+        "text": "var/infection.log",
+        "summary": "var/summary.log",
+        "json": "var/infection-log.json",
+        "perMutator": "var/per-mutator.md",
+        "github": true,
+        "badge": {
+            "branch": "2.0.x"
+        }
+    },
+    "mutators": {
+        "@default": true,
+        "@function_signature": true
```

Comment:
Probably not necessary, when used in combination with the static analysis plugin